.github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md

```diff
@@ -0,0 +1,23 @@
+### Description:
+
+<!--
+This should be a brief one or two line description of the PR. Details should be contained in commit messages.
+-->
+
+---
+
+### Submitter:
+
+Have you done the following?:
+
+* [ ] Added a new directory in the project root with a `Dockerfile` inside?
+* [ ] Tested the image works?
+
+---
+
+### Reviewer:
+
+Have you done the following?:
+
+* [ ] Checked the `Dockerfile` is only installing the necessary packages?
+* [ ] Checked the permissions of downloaded files onto the container are appropriate?
```

Comment:
Could we also add some checks for the reviewer:

- Does the `Dockerfile` switch to `NB_UID` at the end of the file? (i.e the image is run as user, **not** root)
- Has the image been built successfully and pushed to Harbor?

Reply:
Done